### PR TITLE
Update ansible docs according to docs.theforeman.org/nightly urls

### PR DESCRIPTION
**Pull request by girijaasoni**
Note: The satellite URLs will work for 6.15 or higher :)

Blocked by https://github.com/theforeman/foreman_ansible/pull/711

### CODE REVIEW

File: lib/foreman_theme_satellite/documentation.rb [hunk 1]
```diff
@@ -5,8 +5,8 @@ module ForemanThemeSatellite
   module Documentation
     USER_GUIDE_DICTIONARY = {
       # Ansible
-      'ImportingRoles' => "#{ForemanThemeSatellite.documentation_root}/managing_configurations_using_ansible_integration_in_red_hat_satellite/getting_started_with_ansible_in_satellite_ansible#Importing_Ansible_Roles_and_Variables_ansible",
-      'Variables' => "#{ForemanThemeSatellite.documentation_root}/managing_configurations_using_ansible_integration_in_red_hat_satellite/getting_started_with_ansible_in_satellite_ansible#Overriding_Ansible_Variables_in_satellite_ansible",
```
Comment (Lennonka) [inline lib/foreman_theme_satellite/documentation.rb]: I'm not sure if it's a good idea to replace these keys. It's possible that they are coded elsewhere as identifiers and if we remove them, then we lose the connection. But I'll try to find someone who really understands it to ack this.

File: lib/foreman_theme_satellite/documentation.rb [hunk 2]
```diff
@@ -5,8 +5,8 @@ module ForemanThemeSatellite
...
+      'Importing_Ansible_Roles_and_Variables_ansible' => "#{ForemanThemeSatellite.documentation_root}/managing_configurations_using_ansible_integration_in_red_hat_satellite/getting_started_with_ansible_in_satellite_ansible#Importing_Ansible_Roles_and_Variables_ansible",
```
Comment (Lennonka) [inline lib/foreman_theme_satellite/documentation.rb]:
```suggestion
      'Importing_Ansible_Roles_and_Variables_ansible' => "#{ForemanThemeSatellite.documentation_root}/managing_configurations_using_ansible_integration/getting_started_with_ansible_in_satellite_ansible#Importing_Ansible_Roles_and_Variables_ansible",
```

File: lib/foreman_theme_satellite/documentation.rb [hunk 3]
```diff
@@ -5,8 +5,8 @@ module ForemanThemeSatellite
...
+      'Overriding_Ansible_Variables_in_foreman_ansible' => "#{ForemanThemeSatellite.documentation_root}/managing_configurations_using_ansible_integration_in_red_hat_satellite/getting_started_with_ansible_in_satellite_ansible#Overriding_Ansible_Variables_in_satellite_ansible",
```
Comment (Lennonka) [inline lib/foreman_theme_satellite/documentation.rb]:
```suggestion
      'Overriding_Ansible_Variables_in_foreman_ansible' => "#{ForemanThemeSatellite.documentation_root}/managing_configurations_using_ansible_integration/getting_started_with_ansible_in_satellite_ansible#Overriding_Ansible_Variables_in_satellite_ansible",
```

**Review by Lennonka — CHANGES_REQUESTED** [review]
We shortened the guide URL in 6.15, but the long variant is valid for 6.14. Ie. if cherry-picked, it won't work.

**Review by ekohl — CHANGES_REQUESTED** [review]
When you're taking the suggestions in https://github.com/theforeman/foreman_ansible/pull/711 I think you'll need to override `docs_url` in the `LinksController`. We haven't don't that yet, but it needs to happen.

**Comment by ShimShtein** [comment]
@ekohl 

> When you're taking the suggestions in https://github.com/theforeman/foreman_ansible/pull/711 I think you'll need to override docs_url in the LinksController. We haven't don't that yet, but it needs to happen.

We already have override for the `docs_url`: https://github.com/RedHatSatellite/foreman_theme_satellite/blob/4fa1231fe68303389345d21dce0a81eee7fab030/app/controllers/concerns/documentation_controller_branding.rb#L37

Although, once the PR gets in, we will need to add `Managing_Configurations_Ansible` and the chapter names to  [`DOCS_GUIDES_LINKS`](https://github.com/RedHatSatellite/foreman_theme_satellite/blob/4fa1231fe68303389345d21dce0a81eee7fab030/lib/foreman_theme_satellite/documentation.rb#L84)  dictionary.

**Comment by ekohl** [comment]
I know why I couldn't find it: I had the master branch checked out, not develop. My bad

**Review by Lennonka — COMMENTED** [review]
The Ansible changes look alright. I don't have an instance to test them, though.

File: lib/foreman_theme_satellite/documentation.rb [hunk 4]
```diff
@@ -82,8 +82,13 @@ module Documentation
         'Products_and_Repositories_content-management' => "#{ForemanThemeSatellite.documentation_root}/managing_content/importing_content_content-management#Products_and_Repositories_content-management",
       },
       'Managing_Hosts' => {
-        'registering-a-host_managing-hosts' => "#{ForemanThemeSatellite.documentation_root}/managing_hosts/registering_hosts_to_server_managing-hosts#Registering_Hosts_by_Using_Global_Registration_managing-hosts",
-      }
+        'registering-a-host_managing-hosts' => "#{ForemanThemeSatellite.documentation_root}/managing_hosts/registering_hosts_to_server_managing-hosts#Registering_Hosts_managing-hosts",
```
Comment (Lennonka) [inline lib/foreman_theme_satellite/documentation.rb]: Is this change necessary? If it's help for global registration, we should probably prefer the link to global registration directly.

Comment (girijaasoni) [reply lib/foreman_theme_satellite/documentation.rb]: My bad, that happened accidentally

File: lib/foreman_theme_satellite/documentation.rb [hunk 5]
```diff
@@ -83,7 +83,12 @@ module Documentation
       },
       'Managing_Hosts' => {
         'registering-a-host_managing-hosts' => "#{ForemanThemeSatellite.documentation_root}/managing_hosts/registering_hosts_to_server_managing-hosts#Registering_Hosts_by_Using_Global_Registration_managing-hosts",
-      }
+      },
+      'Managing_Configurations_Ansible' =>
+       {
```
Comment (ekohl) [inline lib/foreman_theme_satellite/documentation.rb]:
```suggestion
      'Managing_Configurations_Ansible' => {
```

File: lib/foreman_theme_satellite/documentation.rb [hunk 6]
```diff
@@ -83,7 +83,12 @@ module Documentation
...
+         'Importing_Ansible_Roles_and_Variables_ansible' => "#{ForemanThemeSatellite.documentation_root}/managing_configurations_using_ansible/getting_started_with_ansible_in_satellite_ansible_integration#Importing_Ansible_Roles_and_Variables_ansible",
+         'Overriding_Ansible_Variables_in_foreman_ansible' => "#{ForemanThemeSatellite.documentation_root}/managing_configurations_using_ansible_integration/getting_started_with_ansible_in_satellite_ansible#Overriding_Ansible_Variables_in_satellite_ansible",
+       }
```
Comment (ekohl) [inline lib/foreman_theme_satellite/documentation.rb]:
```suggestion
      }
```

**Review by ekohl — CHANGES_REQUESTED** [review]

File: lib/foreman_theme_satellite/documentation.rb [hunk 7]
```diff
@@ -83,6 +83,11 @@ module Documentation
       },
       'Managing_Hosts' => {
         'registering-a-host_managing-hosts' => "#{ForemanThemeSatellite.documentation_root}/managing_hosts/registering_hosts_to_server_managing-hosts#Registering_Hosts_by_Using_Global_Registration_managing-hosts",
+      },
+      'Managing_Configurations_Ansible' =>
+      {
```
Comment (ekohl) [inline lib/foreman_theme_satellite/documentation.rb]:
```suggestion
      'Managing_Configurations_Ansible' => {
```

File: lib/foreman_theme_satellite/documentation.rb (hunk 1 above)
Comment (ekohl) [reply lib/foreman_theme_satellite/documentation.rb]: It was correct to remove them, because they were removed from the source code.

@girijaasoni please remove the lines again.

**Review by ShimShtein — APPROVED** [review]

**Comment by ShimShtein** [comment]
@ekohl LGTM, any last comments?

**Review by ekohl — APPROVED** [review]
